ui/components/atoms/A004-AppBar/index.js

```diff
@@ -1,17 +1,13 @@
 import React from 'react'
 import createPropTypes from 'json-schema-prop-types'
-import { AppBar, Toolbar, Typography } from '@material-ui/core'
+import { AppBar, Toolbar } from '@material-ui/core'
 
 import schema from './schema'
 
 const A004 = props => {
   return (
-    <AppBar color='primary' position='static'>
-      <Toolbar>
-        <Typography variant='title' color='inherit'>
-          {props.children}
-        </Typography>
-      </Toolbar>
+    <AppBar color='primary' position={props.position}>
+      <Toolbar variant='regular' />
```

Comment:
without the {props.children} in toolbar we wont be able to use this component.

You will need to add it back

Reply:
oh yeah! i will put it back.. Thanks!